cmd/manager/main.go

```diff
@@ -165,6 +165,10 @@ func serveCRMetrics(cfg *rest.Config) error {
 	if err != nil {
 		return err
 	}
+	// To bypass ugly metrics logs after operator-sdk 0.12.0 upgrade
+	// https://github.com/operator-framework/operator-sdk/issues/1858#issuecomment-548323725
+	operatorNs = ""
```

Comment:
I don't get why we get the namespace few lines above.

Reply:
You right, let's remove it all together.